docs/_includes/mailing-list.html

```diff
@@ -0,0 +1,20 @@
+<section class="mailing-section">
+
+    <div class="mailinglist-main">
+        <div class="mailinglist-text">
+            <span>Mailing Lists</span>
+            <p>Engage in the GetNightHawk project. Join any of the mailing lists.</p>
+        </div>
+    
+        
+        <div class="mailinglist-buttons">
```

Comment:
Do a simple center alignment of all three buttons for the mobile view.

Reply:
you means to remove flex right?